reference/shipping.v2.yml

```diff
@@ -1082,12 +1082,12 @@ paths:
         
         | Property | Type | Description |
         |:---------|:-----|:------------|
-        | default_cost | number | Default shipping cost, applied either as a percentage of the orderʼs total value or as a fixed amount. |
+        | default_cost | number &#124; null | Default shipping cost, applied either as a percentage of the orderʼs total value or as a fixed amount. A value of null can be supplied if no default cost is required. |
```

Comment:
For context the `&#124;` is used to display the pipe (`|`) character.

When I tried to escape the `|` character, it still registered the pipe character as another column in the table and the escape char shows

![escape_char](https://github.com/bigcommerce/api-specs/assets/12403409/fb60e773-cbf1-43cc-82af-808625511ecb)